ui/component/or-configuration/src/style.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty file, or maybe you were planning to move some css in here. Just mentioning. ;)